recipes/assetid/meta.yaml

```diff
@@ -0,0 +1,38 @@
+{% set name = "assetid" %}
+{% set version = "0.0.4" %}
+{% set build = 0 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  git_rev: {{ version }}
+  git_url: "https://github.com/axiom-data-science/{{ name }}"
```

Comment:
We prefer tarball instead of a git tag. Can you change this to:

```yaml
fn: assetid-{{ version}}.tar.gz
url: https://github.com/axiom-data-science/assetid/archive/v{{ version }}.tar.gz
sha256: dcd45cd619d2b747e2b895b0b37306fca1369a6220a57d3a680fed2037e28a24
```